.github/workflows/add-prs-and-issues-to-project.yml

```
@@ -0,0 +1,21 @@
+name: Adds all new issues and PRs to the Transition Monitor project
+
+on:
+  issues:
+    types:
+      - opened
+      - transferred
+  pull_request:
+    branches: [main]
+    types:
+      - opened
+
+jobs:
+  add-to-project:
+    name: Add issue to project
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/add-to-project@v0.4.0
+        with:
+          project-url: https://github.com/orgs/RMI-PACTA/projects/8
+          github-token: ${{ secrets.CJ_ADD_TO_PROJECT_PAT }}
```

```suggestion
          github-token: ${{ secrets.GITHUB_TOKEN }}
```

So we don't rely on any one person's PATs? 

https://docs.github.com/en/actions/security-guides/automatic-token-authentication

---

```suggestion
          github-token: ${{ secrets.PAT_ADD_ISSUES_TO_PROJECT }}
```

I've created the @PACTA-bot user who created the `PAT_ADD_ISSUES_TO_PROJECT` PAT with `repo` and `project` scopes.